src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -111,5 +111,11 @@
       <Name>VoidMainWithExitCodeApp</Name>
     </ProjectReference>
   </ItemGroup>
+  <!-- ToDo: Remove this target once the infrastructure for testing in netcoreapp1.1 is ready -->
+  <Target Name="RemoveUnwantedTestMonikers" Condition="'$(TargetGroup)'=='netstandard1.7'" BeforeTargets="CopyTestToTestDirectory">
```

Comment:
yeah that makes sense. Do you want that to happen as part of this PR or if not then I'll submit a new PR moving all of these to dir.targets once this is merged.